config/serverless.chat.yml

```diff
@@ -35,8 +34,8 @@ xpack.legacy_uptime.enabled: false
 xpack.ux.enabled: false
 xpack.search.enabled: false
 xpack.searchAssistant.enabled: false
-xpack.search.connectors.enabled: false
-xpack.search.homepage.enabled: false
```

Comment:
These settings disabled the correct plugins, but then failed when validating the config against the `xpack.search` path.